docs/UserGuide.md

```diff
@@ -224,6 +223,22 @@ Shows a list of all shifts in the McScheduler, including the roles needed and wo
 
 Format: `shift-list`
 
+### Locating shift by day or time: `shift-find`
+
+Finds shifts whose day or time contain any of the given keywords.
+
+Format: `shift-find KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g. `mon` will match `MON`
+* Only the day and time is searched.
+* Only full words will be matched e.g. `m` will not match `MON`
+* Workers matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `MON AM` will return a `TUE AM` shift and a `MON PM` shift
+
+Examples:
+* `shift-find Mon` returns a `MON AM` shift and a `MON PM` shift
+* `shift-find AM PM` returns all shifts
```

Comment:
```suggestion
* `shift-find Fri PM` returns a `FRI PM` shift

* `shift-find Mon` returns a `MON AM` shift and a `MON PM` shift

* `shift-find AM PM` returns all shifts
```
Perhaps we can add an example which searches both the day and time

Reply:
Adjusted the suggest to reflect functionality as Fri PM returns all Fri and all PM shifts (don't think need a FRI PM specific search as Fri will at most return 2)